content/articles/support-vector-regression-in-python/index.md

```diff
@@ -0,0 +1,303 @@
+### Getting Started with Support Vector Regression in Python
+Support Vector Regression, denoted as SVR, is a regression function that generalizes the finite Support Vector Machines model on a continuous-valued output. Although less popular in real-valued function estimation, this algorithm has proven very effective.
```

Comment:
`that generalizes the finite Support Vector Machines model on a continuous-valued output`

This statement is not easily understood by the beginners unless they have some idea about SVM. So, add 1 - 2 lines of explanation about it.

Reply:
Also, in the intro make sure to add why we are learning about SVR, and what are we going to learn in this article.